apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -65,6 +67,19 @@ export class SendMessageInApp extends SendMessageBase {
 
   @InstrumentUsecase()
   public async execute(command: SendMessageCommand) {
+    await this.executionDetailsArchiveProducer.add(
```

Comment:
![image](https://github.com/novuhq/novu/assets/39195835/5db058b4-98d0-48f5-a87b-95c676e221f7)

Reply:
definitely for now i created a minimalistic solution only for us to see how it could look.